src/client/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -346,9 +344,12 @@ public PublicClientApplicationBuilder WithKerberosTicketClaim(string servicePrin
         /// <item><description>In .NET 6 apps, target <c>net6.0-windows10.0.17763.0</c> for all Windows versions and target <c>net6.0</c> for Linux and Mac.</description></item>
         /// <item><description>In .NET classic or .NET Core 3.1 apps, install Microsoft.Identity.Client.Desktop first and call <c>WithDesktopFeatures()</c>.</description></item>
         /// <item><description>In mobile apps, the device must be Intune joined and Authenticator or Company Portal must be installed. See https://aka.ms/msal-brokers </description></item>
+        /// This method is deprecated for mobile platforms.
         /// </list>
         /// </remarks>
+#if ANDROID || iOS || WINDOWS_APP || MAC

```

Comment:
Don't think MAC is needed here; it'll be covered under [iOS flag](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/c5b0106d04298698f84fa434b44055023912d2cf/build/platform_and_feature_flags.props#L24).